_pages/en/strategy-build-vs-buy.md

```diff
@@ -28,510 +26,493 @@ th {
-
-As such, if the organization is not in a position to design, build and operate the solution itself, the third party putting the solution together for the benefit of ESDC should have clear directives to align with our own architectural principles.
-
-### Buy
-
-The buy decision should be reserved for when the needs are non-domain specific, repeatable, well-defined, and unlikely to change due to legislative changes.
-
-During a buy decision, assessment factors should include:
-
-- degree of customization required (significant customization suggests a build decision is more appropriate)
-- exit costs and exit challenges,
-- whether the vendor will agree to the GC Terms and Conditions.
-
-The department should not offload its responsibility to Canadians through offloading risk to vendors.
-Furthermore, through large purchases the government is outsourcing expertise required to deliver on its mandate.
-Lastly, doing so introduces additional context switches (or handoffs) between organizations, reducing the government's agility to implement its own policy changes, and limits its ability to respond to emergency situations.
-
-As per the [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602), when buy decisions are made, they will prioritize Software as a Service (SaaS) over purchases requiring the maintenance of physical, Platform as a Service (PaaS) or Infrastructure as a Service (IaaS).
-When selecting which cloud services to leverage, the [Government of Canada Right Cloud Selection Guidance](https://www.canada.ca/en/government/system/digital-government/modern-emerging-technologies/cloud-services/government-canada-right-cloud-selection-guidance.html) will be adhered to.
-Further consideration should be made with respect to the softwares use of Open Source Software (OSS).
-When considering COTS, the [Software Licencing Supply Arrangement](https://www.tpsgc-pwgsc.gc.ca/app-acq/cral-sarc/lgcl-ctgr-eng.html) will be used.
-Preference shall be given in the following order[^1] in addition to the aforementioned cloud prioritization:
-
-1. OSS using open standards
-1. OSS
-1. OSS + Closed COTS using open standards
-1. OSS + Closed COTS - (be aware of lock-in)
-1. Closed COTS using open standards - (be aware of lock-in)
-1. Closed COTS - (be aware of lock-in)
-
-#### Risk Managing the Obtention of Software
-
-The Strategy looks at risk managing the obtention of software over following dogmatic rules to purchase or build software (e.g. "SaaS first" does not mean SaaS always, "buy what we can" does not mean buying because it's available).
-Instead, the Strategy will use such policy instruments as intended by the policy author and its authority to inform decisions for long term sustainability and user needs.
-
-The risk management for software acquisition is based on the following formula:
```

Comment:
Do we still want this line to say 'acquisition' or should it be changed to obtention?  Nice word by the way (obtention).  I had to look it up! :-)